Stack/Opc.Ua.Core/Stack/Server/EndpointBase.cs

```diff
@@ -848,6 +939,19 @@ private void OnProcessRequest(object state)
                     // set the context.
                     SecureChannelContext.Current = m_context;
 
+                    // check if AdditionalHeader field from the request contains trace information
+                    if (m_request.RequestHeader.AdditionalHeader != null && m_request.RequestHeader.AdditionalHeader.Body is AdditionalParametersType parameters)
```

Comment:
always create the activity, but only add the parent if provided in the header.

Reply:
updated